.ci_support/linux_python3.6.yaml

```diff
@@ -21,7 +21,7 @@ geos:
 gsl:
 - '2.5'
 icu:
-- '64.2'
+- '58.2'
```

Comment:
We need to use the old `icu` until we have new builds of `qt`.
(I'm working on them today.)